hcatalog/server-extensions/src/main/java/org/apache/hive/hcatalog/listener/DbNotificationListener.java

```diff
@@ -1142,10 +1142,8 @@ public void run() {
         } catch (Exception ex) {
           //catching exceptions here makes sure that the thread doesn't die in case of unexpected
           //exceptions
-          LOG.warn(
-              "Exception received while cleaning notifications. More details can be found in debug mode"
-                  + ex.getMessage());
-          LOG.debug(ex.getMessage(), ex);
```

Comment:
Why this debug log is removed? Not sure, if it's there for any security issues.

Reply:
There is no point in dropping a warning with just the exception message and asking user to turn on debug level to investigate further. An exception has happened and if we are giving out "warn"ing better we also log the exception accompanying it instead of a separate debug message. WARN has higher priority than debug.

It will make the logs slightly verbose but will be more useful in production environment where changing log level isn't that easy.